src/randoop/sequence/ExecutableSequence.java

```diff
@@ -860,4 +863,32 @@ public static int checks_count (List<ExecutableSequence> seqs) {
 
     return cnt;
   }
+
+  /**
+   * initialize - used by ExecutionVisitors
```

Comment:
What does it do?
(Knowing how it is supposed to be used is useful but not enough.)